### ISSUE TITLE
Failing migration to v2.0.0-alpha.2

### ISSUE THREAD

**Issue by mzbroch** [nautobot/nautobot#3518]
### Environment
* Nautobot version: v2.0.0-alpha.2
* Python version: 3.7
* Database platform, version: Postgres
* Middleware(s):

### Steps to Reproduce
1. Import existing 1.5.14 dataset into Postgres
2. Run `invoke debug`
3.

### Expected Behavior

Migrations work

### Observed Behavior

Migrations fail with following output:
```
nautobot_1       |   Applying extras.0060_role_and_alter_status... OK
db_1             | 2023-03-30 09:48:38.151 UTC [129] ERROR:  duplicate key value violates unique constraint "extras_role_slug_key"
db_1             | 2023-03-30 09:48:38.151 UTC [129] DETAIL:  Key (slug)=(loopback) already exists.
db_1             | 2023-03-30 09:48:38.151 UTC [129] STATEMENT:  INSERT INTO "extras_role" ("id", "created", "last_updated", "_custom_field_data", "name", "slug", "color", "description", "weight") VALUES ('5f4d5a5a-0cb9-44cc-bad8-b4a9c9c95863'::uuid, '2023-03-30'::date, '2023-03-30T09:48:38.150962+00:00'::timestamptz, '{}', 'Loopback', 'loopback', '9e9e9e', '', NULL)
nautobot_1       |   Applying extras.0061_collect_roles_from_related_apps_roles...Traceback (most recent call last):
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
nautobot_1       |     return self.cursor.execute(sql, params)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django_prometheus/db/common.py", line 71, in execute
nautobot_1       |     return super().execute(*args, **kwargs)
nautobot_1       | psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "extras_role_slug_key"
nautobot_1       | DETAIL:  Key (slug)=(loopback) already exists.
nautobot_1       |
nautobot_1       |
nautobot_1       | The above exception was the direct cause of the following exception:
nautobot_1       |
nautobot_1       | Traceback (most recent call last):
nautobot_1       |   File "/usr/local/bin/nautobot-server", line 6, in <module>
nautobot_1       |     sys.exit(main())
nautobot_1       |   File "/source/nautobot/core/cli.py", line 61, in main
nautobot_1       |     initializer=_configure_settings,  # Called after defaults
nautobot_1       |   File "/source/nautobot/core/runner/runner.py", line 263, in run_app
nautobot_1       |     management.execute_from_command_line([runner_name, command] + command_args)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
nautobot_1       |     utility.execute()
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 413, in execute
nautobot_1       |     self.fetch_command(subcommand).run_from_argv(self.argv)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 354, in run_from_argv
nautobot_1       |     self.execute(*args, **cmd_options)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 398, in execute
nautobot_1       |     output = self.handle(*args, **options)
nautobot_1       |   File "/source/nautobot/core/management/commands/post_upgrade.py", line 82, in handle
nautobot_1       |     verbosity=options["verbosity"],
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 181, in call_command
nautobot_1       |     return command.execute(*args, **defaults)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 398, in execute
nautobot_1       |     output = self.handle(*args, **options)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 89, in wrapped
nautobot_1       |     res = handle_func(*args, **kwargs)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/migrate.py", line 246, in handle
nautobot_1       |     fake_initial=fake_initial,
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py", line 117, in migrate
nautobot_1       |     state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
nautobot_1       |     state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py", line 227, in apply_migration
nautobot_1       |     state = migration.apply(state, schema_editor)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/migrations/migration.py", line 126, in apply
nautobot_1       |     operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
nautobot_1       |     self.code(from_state.apps, schema_editor)
nautobot_1       |   File "/source/nautobot/extras/migrations/0061_collect_roles_from_related_apps_roles.py", line 122, in populate_roles_from_related_app_roles
nautobot_1       |     create_equivalent_roles_of_ipaddress_role_choices(apps)
nautobot_1       |   File "/source/nautobot/extras/migrations/0061_collect_roles_from_related_apps_roles.py", line 57, in create_equivalent_roles_of_ipaddress_role_choices
nautobot_1       |     bulk_create_roles(apps, roles_to_create, ["ipam.ipaddress"])
nautobot_1       |   File "/source/nautobot/extras/migrations/0061_collect_roles_from_related_apps_roles.py", line 93, in bulk_create_roles
nautobot_1       |     new_role.save()
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py", line 740, in save
nautobot_1       |     force_update=force_update, update_fields=update_fields)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py", line 778, in save_base
nautobot_1       |     force_update, using, update_fields,
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py", line 881, in _save_table
nautobot_1       |     results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/base.py", line 921, in _do_insert
nautobot_1       |     using=using, raw=raw,
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/manager.py", line 85, in manager_method
nautobot_1       |     return getattr(self.get_queryset(), name)(*args, **kwargs)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/query.py", line 1270, in _insert
nautobot_1       |     return query.get_compiler(using=using).execute_sql(returning_fields)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/models/sql/compiler.py", line 1416, in execute_sql
nautobot_1       |     cursor.execute(sql, params)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 98, in execute
nautobot_1       |     return super().execute(sql, params)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/cacheops/transaction.py", line 97, in execute
nautobot_1       |     result = self._no_monkey.execute(self, sql, params)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 66, in execute
nautobot_1       |     return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
nautobot_1       |     return executor(sql, params, many, context)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
nautobot_1       |     return self.cursor.execute(sql, params)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/utils.py", line 90, in __exit__
nautobot_1       |     raise dj_exc_value.with_traceback(traceback) from exc_value
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
nautobot_1       |     return self.cursor.execute(sql, params)
nautobot_1       |   File "/usr/local/lib/python3.7/site-packages/django_prometheus/db/common.py", line 71, in execute
nautobot_1       |     return super().execute(*args, **kwargs)
nautobot_1       | django.db.utils.IntegrityError: duplicate key value violates unique constraint "extras_role_slug_key"
nautobot_1       | DETAIL:  Key (slug)=(loopback) already exists.
```

**Comment by jathanism** [comment]
This is a known issue but thanks for submitting it so we can track it! :)

**Comment by glennmatthews** [comment]
Fixed in #3596.
